cumulus_library/studies/core/core_templates/medicationrequest.sql.jinja

```diff
@@ -0,0 +1,59 @@
+{% import 'core_utils.jinja' as utils %}
+CREATE TABLE core__medicationrequest AS
+WITH temp_mr AS (
+    SELECT 
+        {{- utils.basic_cols(
+                'medicationrequest',
+                'mr',
+                ['id', 'status', 'intent']
+            ) 
+        }},
+        {{- utils.date_cols_from_str(
```

Comment:
tiny nit: this method and `truncate_date_cols` is starting to feel redundant - is there an opportunity to merge these two into the same - like allow a non-period-tuple argument that just uses the raw date? But absolutely not necessary - I'm just trying to make your `utils.jinja` file even longer :smile:

Reply:
I'm going to pass on this for now - these are enough different in context to me that i want them to be explicit.